changelog.json

```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[Fixed] Kill off unnecessary margin inside compare tab branch list - #6058",
```

Comment:
This entry and the one below were also related to a regression that hasn't hit production yet. Let's omit them (yeah, this means it's an empty changelog now but that's okay).